src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings

```diff
@@ -195,3 +195,19 @@
 "ExposureSubmission_Antigen_Profile_Legal_Text06" = "Der QR-Code enthält Ihre persönlichen Angaben. Stellen Sie Ihren persönlichen QR-Code niemandem zur Verfügung, wenn Sie nicht wollen, dass die Daten ausgelesen werden. Wir empfehlen Ihren persönlichen QR-Code nicht zu veröffentlichen und nicht per E-Mail zu versenden.";
 
 "ExposureSubmission_Antigen_Profile_Legal_Text07" = "Private Personen oder Unternehmen dürfen von Ihnen nicht verlangen, dass Sie Ihr Schnelltest-Profil vorzeigen.";
+
+/* Health Certificate Information */
+
+"HealthCertificate_Info_Legal_headline" = "Ihr Einverständnis";
+
+"HealthCertificate_Info_Legal_subHeadline" = "Durch Antippen von „Einverstanden“ willigen Sie in folgende Schritte ein:";
+
+"HealthCertificate_Info_Legal_section01" = "Die App legal";
```

Comment:
Wat? :D

Reply:
the legal texts are not final at the moment

Reply:
Ah okay 👍